liveblog/liveblog/settings.py

```diff
@@ -44,6 +44,11 @@
 
 ROOT_URLCONF = 'liveblog.urls'
 
+import os
+redis_host = "localhost"
+if os.environ['WITH_DOCKER'] == 'True':
```

Comment:
This will fail if the environment variable isn't present. I suggest changing it to have a REDIS_HOST env var and then just use `os.environ.get('REDIS_HOST', 'localhost')` directly in the CONFIG section.